project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
```

Comment:
I needed to upgrade sbt version for this fix: https://github.com/sbt/sbt/pull/3265

There is a platform-specific issue that comes up for some people with newer versions of ncurses: https://github.com/sbt/sbt/issues/3240

Let me know if this should be part of a separate PR.